specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10340 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ClassificationRules_ListAll": {
+            "$ref": "./examples/ClassificationRules_ListAll.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/classificationrules/{classificationRuleName}/versions": {
+      "get": {
+        "tags": [
+          "ClassificationRules"
+        ],
+        "description": "Lists the rule versions of a classification rule.",
+        "operationId": "ClassificationRules_ListVersionsByRuleName",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "classificationRuleName",
+            "required": true,
+            "type": "string",
+            "description": "The name of the classification rule to retrieve."
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/ClassificationRuleList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ClassificationRules_ListVersionsByRuleName": {
+            "$ref": "./examples/ClassificationRules_ListVersionsByRuleName.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/classificationrules/{classificationRuleName}/versions/{classificationRuleVersion}:tag": {
+      "post": {
+        "tags": [
+          "ClassificationRules"
+        ],
+        "description": "Sets Classification Action on a specific classification rule version.",
+        "operationId": "ClassificationRules_TagClassificationVersion",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "classificationRuleName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "classificationRuleVersion",
+            "required": true,
+            "type": "integer",
+            "description": "The version of the classification rule to retrieve.",
+            "format": "int32"
+          },
+          {
+            "classificationAction": {
+              "type": "string",
+              "description": "The classification action type.",
+              "$ref": "#/definitions/ClassificationAction"
+            }
+          },
```

Comment:
This is invalid OpenAPI. This must be a parameter object, not a property.

Reply:
Yes, I have changed it to object